doc/sphinx-guides/source/installation/config.rst

```diff
@@ -709,245 +661,199 @@ While the above two options are recommended because they have been tested by the
 
 For example, the Australian Data Archive (ADA) successfully uses the Australian National Data Service (ANDS) API (a proxy for DataCite) to mint their DOIs through Dataverse using a ``doi.baseurlstring`` value of "https://researchdata.ands.org.au/api/doi/datacite" as documented at https://documentation.ands.org.au/display/DOC/ANDS+DataCite+Client+API . As ADA did for ANDS DOI minting, any DOI provider (and their corresponding DOI configuration parameters) other than DataCite must be tested with Dataverse to establish whether or not it will function properly.
```

Comment:
Couple lines above there is `<jvm-options>-Ddoi.username=...</jvm-options>` which we no longer support